### PR TITLE
Digitigrade Paralysis BugFix

### PR DESCRIPTION
**Pull request by SpaceDragon00**
## About The Pull Request

- A limb's attach_limb proc now uses the set_owner() proc instead of setting the owner manually, fixing problems where limbs that were meant to be paralyzed were not
- Added a sanity check to set_usable_legs to ensure the value isn't set lower than 0

## Why It's Good For The Game

Fewer bugs, more sanity

## Changelog
:cl:
fix: People with digitigrade legs are now properly paralyzed by the Paraplegic perk
fix: Usable_limbs can no longer be set below 0, which let people walk when they shouldn't be able to
/:cl:

### CODE REVIEW

File: code/modules/surgery/bodyparts/dismemberment.dm [hunk 1]
```diff
@@ -323,7 +323,7 @@
 		return FALSE
 	. = TRUE
 	moveToNullspace()
-	owner = new_limb_owner
```
Comment (LemonInTheDark) [inline code/modules/surgery/bodyparts/dismemberment.dm:326]: Ah this was my fault. wonderful

File: code/modules/mob/living/living.dm [hunk 2]
```diff
@@ -1819,6 +1819,9 @@
 /mob/living/proc/set_usable_legs(new_value)
 	if(usable_legs == new_value)
 		return
+	if(new_value < 0) // Sanity check
+		new_value = 0
+
```
Comment (LemonInTheDark) [inline code/modules/mob/living/living.dm:1824]: Don't you think we should error on this?

**Comment by MrMelbert** [comment]
Fixes #56141

File: code/modules/mob/living/living.dm (hunk 2 above)
Comment (SpaceDragon00) [reply code/modules/mob/living/living.dm:1824]: Good idea, I'll add that real quick

**Comment by Time-Green** [comment]
There go the days of having my legs replaced to walk again

**Review by LemonInTheDark — APPROVED** [review]

**Pull request merged**